scripts/install_test_dependencies.sh

```diff
@@ -15,8 +15,19 @@
 
 set -ex
 
-sudo apt-get update
-sudo apt-get install -y php5-cli
+if [ "${CIRCLECI}" != "true" ]; then
+    sudo apt-get update
+    sudo apt-get install -y php5-cli
+    # Install gcloud
```

Comment:
I wanted to use the latest gcloud sdk, not the preinstalled one. I'm fine with upgrading on demand like I do now.

I prefer using our own installation because potentially I can test whatever version including prerelease, specific version, etc.